### PR TITLE
feat(reporter): log coverage threshold as a warning [#432]

### PR DESCRIPTION
**Pull request by mkhodan**
Please see #432

### CODE REVIEW

**Comment by mkhodan** [comment]
@googlebot I signed it!

**Comment by mkhodan** [comment]
will close #433

File: test/reporter.spec.js [hunk 1]
```diff
@@ -538,5 +538,54 @@ describe('reporter', () => {
 
       expect(results.exitCode).to.equal(0)
     })
+
+    it.only('should log warnings on low coverage and not fail the build', async () => {
```
Comment (anthony-redFox) [inline test/reporter.spec.js:542]: need to remove only

**Review by anthony-redFox — CHANGES_REQUESTED** [review]

**Comment by djgrove** [comment]
Howdy. Is this still on the radar to be added to `karma-coverage`? This is a feature I would be really excited to see as well.

**Comment by djgrove** [comment]
@mkhodan I think @anthony-redFox was waiting on you to address the above comment but there was otherwise no issue with PR. Have you removed the `only` from the above test?

**Comment by mkhodan** [comment]
@djgrove this is already merged, please check #446

**Pull request closed without merging**